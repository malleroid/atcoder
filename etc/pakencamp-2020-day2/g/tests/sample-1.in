5 8
5 1 5
4 5 2
2 5 4
3 5 2
3 2
4 8
2 4
3 8
4 3
1 8
3 10
2 1
