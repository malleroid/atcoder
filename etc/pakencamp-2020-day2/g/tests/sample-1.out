7
19
11
19
9
-1
23
5
